proto/osmosis/lockup/lock.proto

```diff
@@ -54,16 +54,16 @@ message PeriodLock {
 enum LockQueryType {
   option (gogoproto.goproto_enum_prefix) = false;
 
-  ByDuration = 0;
-  ByTime = 1;
+  ByTime = 0;
+  ByDuration = 1;
```

Comment:
Doesn't this break every existing lock and would require a state migration in the upgrade handler?